helm/ingress-controller/scripts/gen-agent-config.sh

```diff
@@ -1,14 +1,11 @@
 #!/bin/bash
 
-NGROK_LOG="${NGROK_LOG:-stdout}"
-NGROK_REGION="${NGROK_REGION:-us}"
```

Comment:
this is hard to test and we are moving away from the agent so i didn't add much here. But what i did do was ran the e2e tests, checked in the dashboard that the edge was defaulted still using US and catted out my agent file in the pod to see its set to empty and the agent is fine with that

```
k exec -it pod/ngrok-ingress-controller-manager-6d7dbd47b7-7gpgb -c ngrok-cli -- /bin/bash

ngrok@ngrok-ingress-controller-manager-6d7dbd47b7-7gpgb:/$ cat /var/lib/ngrok/agent-template.yaml
version: 2
authtoken: REDACTED
console_ui: false
region:
remote_management: true
update_check: false
```